docs/en/Collector-Table-Description.md

```diff
@@ -0,0 +1,572 @@
+application_id | c3 | Integer | Owner application id
+time_bucket | tb | Long | [Date format](Collector-Table-Description.md#Metric-table-time-bucket)
+
+### ApplicationMapping
+- Table name: application_mapping_`TimeDimension`
+- TimeDimension contains minute, hour, day, month
+- For example: A application invoke B application, collector will generate two metrics:
+    * From the caller's trace data: A application -> B application's IP address (Topology will use this metric when B application is not monitored by agent)
+    * From the callee's trace data: A application -> B application (Topology will use this metric when B application is monitored by agent)
+
+Column Name | Short Name | Data Type | Description
+----------- | ---------- | --------- | ---------
+_id | _id | Keyword | primary key, es speciality, `time_bucket`_`metric_id`
+metric_id | c1 | Keyword | `application_id`_`mapping_application_id`
+application_id | c2 | Integer | Registered at server side.
+mapping_application_id | c3 | Integer | Registered at client side with the server's IP address.
+time_bucket | tb | Long | [Date format](Collector-Table-Description.md#Metric-table-time-bucket)
+
+### ApplicationMetric
+- Table name: application_metric_`TimeDimension`
+- TimeDimension contains minute, hour, day, month
+
+Column Name | Short Name | Data Type | Description
+----------- | ---------- | --------- | ---------
+_id | _id | Keyword | primary key, es speciality, `time_bucket`_`metric_id`
+metric_id | c1 | Keyword | `application_id`_`source_value`
+application_id | c2 | Integer | Owner application id
+source_value | c3 | Integer | Caller(0), Callee(1)
+transaction_calls | a1 | Long | The total number of calls, sums values aggregate by `time_bucket`
+transaction_error_calls | a2 | Long | The total number of error calls, sums values aggregate by `time_bucket`
+transaction_duration_sum | a3 | Long | The duration sum of all calls, sums values aggregate by `time_bucket`
+transaction_error_duration_sum | a4 | Long | The duration sum of error calls, sums values aggregate by `time_bucket`
+transaction_average_duration | a5 | Long | The average duration of all calls, used for order by this column in database.
+business_transaction_calls | b1 | Long | 
+business_transaction_error_calls | b2 | Long | 
+business_transaction_duration_sum | b3 | Long | 
+business_transaction_error_duration_sum | b4 | Long | 
+business_transaction_average_duration | b5 | Long | 
+mq_transaction_calls | m1 | Long | 
+mq_transaction_error_calls | m2 | Long | 
+mq_transaction_duration_sum | m3 | Long | 
+mq_transaction_error_duration_sum | m4 | Long | 
+mq_transaction_average_duration | m5 | Long | 
+satisfied_count | d1 | Long | [The formula](../../apm-collector/apm-collector-core/src/main/java/org/apache/skywalking/apm/collector/core/util/ApdexThresholdUtils.java)
```

Comment:
Add `appdex` in description, `appdex` is a common concept in APM field. I think most people are easy to understand that.